test/quantization/test_numeric_suite_fx.py

```diff
@@ -429,3 +443,117 @@ def test_compare_model_outputs_lstm_dynamic_fx(self):
             lstm_input,
             lstm_hidden,
         )
+
+class TestFXGraphMatcher(QuantizationTestCase):
+    # TODO(future PR): more tests
+
+    def test_conv_mod_fp32_prepared_vs_int8(self):
```

Comment:
Can we add test case for matching fused ops like float and quantized ConvRelu in future PRs?

Reply:
sure, will do